HAR-5778 Poista duplikaattiturvalaitteet

Jarzka opened the pull request.

Jarzka committed:
Poista duplikaattiturvalaitteet

Jarzka committed:
Korjaa virhe migraatiossa

Kauko committed:
Merge branch 'develop' into HAR-5778

tatut commented on tietokanta/src/main/resources/db/migration/V1_586__.sql:
```diff
@@ -0,0 +1,6 @@
+WHERE id NOT IN
+      (SELECT MAX(id)
+       FROM vv_turvalaite
+       GROUP BY turvalaitenro);
```
Jos tiettyä turvalaitenumeroa on vain yksi rivi, eikö tämä poista senkin?

tatut committed:
Tee turvalaitenumerosta uniikki

tatut committed:
Upsert turvalaitenro perusteella

Käytetään turvalaitenro kenttää upsertin tekemiseen. Tunniste näyttää
olevan random generoitua ja muuttuu joka kerta. Pudotetaan se kokonaan
pois päivityksestä.

tatut committed:
Poista turvalaitteen tunniste

Kenttä on täysin turha, koska se muuttuu joka haulla (random
generoituu inspiren jsoniin)

tatut committed:
Rename V1_586__.sql to V1_587__.sql

tatut committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into HAR-5778

tatut committed:
Muuta ns alias

tatut committed:
Poista tunnisteet testidatasta

tatut committed:
Korjaa turvalaitteen liittäminen triggerissä

Kauko committed:
Merge branch 'develop' into HAR-5778
